test/nieSyntaxSpecs.js

```diff
@@ -0,0 +1,42 @@
+describe('Testing nie syntax services', function() {
+	var $injector;
```

Comment:
You are right. The ideal thing would be to ask for the injector outside the beforeEach method so it would be done just once. You could check it that works fine.